aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -1003,7 +1003,7 @@ module aptos_framework::account {
         // Maul the signature and make sure the call would fail
         let invalid_signature = ed25519::signature_to_bytes(&sig);
         let first_sig_byte = vector::borrow_mut(&mut invalid_signature, 0);
-        *first_sig_byte = *first_sig_byte + 1;
+        *first_sig_byte = *first_sig_byte ^ 1;
```

Comment:
Is ^ XOR? What was wrong with +?

Reply:
yes it's XOR. byte + 1 may overflow. but XOR won't.